lib/util/pools.js

```diff
@@ -78,10 +78,8 @@ export function createPool(name, opts) {
       }
     },
 
-    freeAll: function() {
-      let allocatedLength = allocated.length;
-      let freeLength = free.length;
-      let reAlloc = allocated.slice(0, size - freeLength);
+    freeAll: function freeAllPool() {
+      let reAlloc = allocated.slice(0, size - free.length);
```

Comment:
Should this be `free.length - 1`? Not 100% sure what `free` and `size` represent, but I figured I'd point it out since I often need to subtract 1 when going from lengths to zero-based indices.